frontend/src/config/patient.schema.json

```diff
@@ -0,0 +1,145 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "patient.schema.json",
+    "title": "Patient",
+    "description": "A patient record for COVID testing",
+    "type": "object",
+    "properties": {
+      "patientID": {
+        "description": "Unique patient identifier",
+        "type": "string"
+      },
+      "patientLastName": {
+        "description": "Patient's last name",
+        "type": "string"
+      },
+      "patientFirstName": {
+        "description": "Patient's first name",
+        "type": "string"
+      },
+      "patientMiddleName": {
+        "description": "Patient's middle name",
+        "type": "string"
+      },
+      "patientSuffix": {
+        "description": "",
+        "type": "string"
+      },
+      "patientRace": {
+        "description": "Patient race",
+        "type": "string"
+      },
+      "patientDOB": {
+        "description": "Patient date of birth",
+        "type": "string",
+        "format": "date"
+      },
+      "patientGender": {
+        "description": "Patient sex",
+        "type": "string"
+      },
+      "patientEthnicity": {
+        "description": "Patient ethnicity",
+        "type": "string"
+      },
+      "patientStreet": {
```

Comment:
thoughts on having Address be its own object, so that you can reference it in other places in a schema where we might need address and apply the same address field/formatting logic?

Reply:
This makes sense - there's at least a couple different address blocks in other schemas. Maybe `patientAddress` then the block labels are just `street` `street2` `city` etc.?

Reply:
pushed a commit for this